modules/ximgproc/src/quaternion.cpp

```diff
@@ -0,0 +1,206 @@
+﻿#include "precomp.hpp"
+using namespace std;
+using namespace cv;
```

Comment:
`using namespace cv;` is not needed. All code is placed into `cv` namespace.

Reply:
OK

Reply:
OK